apps/org.videolan.VLC/eos-vlc.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
+Version=1.0
+Name=Get VideoLAN Client
+# Only KDE 4 seems to use GenericName, so we reuse the KDE strings.
+# From Ubuntu's language-pack-kde-XX-base packages, version 9.04-20090413.
+GenericName=Media Player
+# Gnome and KDE 3 uses Comment.
+Comment=Media Player
+Exec=/usr/bin/eos-install-app-helper --app-id org.videolan.VLC --remote eos-apps --app-name "VideoLAN Client" --old-desktop-file-name eos-vlc.desktop --new-desktop-file-name org.videolan.VLC.desktop --required-archs x86_64 %U
+Terminal=false
+Icon=eos-vlc
+Type=Application
+X-Endless-LaunchMaximized=true
```

Comment:
I would remove that one, as it is misleading to show the speedwaggon of the app when transitioning to gnome-software.